evaluate.py

```diff
@@ -0,0 +1,60 @@
+"""Testing models inference"""
+import sys
+from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter
+
+from transformers import pipeline
+
+from decompile.trainers.llama_trainer import LLaMaTrainer
+
+
+def main() -> int:
+    """Main entry point for inference"""
+
+    parser = ArgumentParser(
+        prog="Evaluate",
+        description="Testing models inference",
+        epilog="Hope it goes well!",
+        formatter_class=ArgumentDefaultsHelpFormatter,
+    )
+
+    parser.add_argument(
+        "--model-path",
+        type=str,
+        required=True,
+        help="Path to the model.",
+    )
+    parser.add_argument(
+        "--tokenizer-path",
+        type=str,
+        required=True,
+        help="Path to the tokenizer.",
+    )
+    parser.add_argument(
+        "--input",
+        type=str,
+        required=True,
+        help="Input assembly.",
+    )
+    parser.add_argument(
+        "--max-length",
+        type=int,
+        default=1024,
+        help="Maximum length of the output.",
+    )
+
+    args = parser.parse_args()
```

Comment:
Maybe making an argparse function that returns args is better.

Reply:
No need to do that now.